test/unit/view/drag-handle.spec.js

```diff
@@ -922,6 +922,23 @@ describe('drag handle', () => {
         expect(event.defaultPrevented).toBe(false);
       });
 
+      it('should cancel when the window is not visibile', () => {
```

Comment:
can this test please be moved to the shared 'control' section at the bottom so that each sensor type can be tested against this functionality?

Reply:
Sure thing, Which section? Is it the `interactive elements` section?